### PR TITLE
ENG-3759: adding i18n support

### PR DESCRIPTION
**Pull request by gReis89**

### CODE REVIEW

**Review by ichalagashvili — APPROVED** [review]
LGTM

File: microfrontends/app-builder-menu/src/components/content.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    emailConfiguration: intl.formatMessage({
+      id: 'app.emailConfiguration',
+      defaultMessage: 'Email Configuration'
+    }),
+    reloadConfiguration: intl.formatMessage({
+      id: 'app.reloadConfiguration',
+      defaultMessage: 'Reload Configuration'
+    }),
+  };
+}
```
Comment (Dv92) [inline microfrontends/app-builder-menu/src/components/content.ts]: please move this file to `src` instead `components`

**Review by Dv92 — COMMENTED** [review]
@gReis89 we need also the translations for the other languages `it` and `pt`

File: microfrontends/app-builder-menu/package-lock.json [hunk 2]
```diff
@@ -1,8 +1,11089 @@
 {
   "name": "app-builder-menu",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (Dv92) [inline microfrontends/app-builder-menu/package-lock.json]: please, use the node version `14.19.2` for installing dependencies in this project. Regenerate this file once you have set the correct version

Comment (gReis89) [reply microfrontends/app-builder-menu/package-lock.json]: done!

**Review by Dv92 — APPROVED** [review]

**Pull request merged**